releases/net472/KnownIssues/695400 - .NET 4.7.2 setup skips installing payload on Win10 machine when upgraded from Win7 with .NET 4.7.2 .md

```diff
@@ -0,0 +1,42 @@
+# .NET 4.7.2 setup fails to install files on OS when upgraded from Windows 7 SP1 or Windows Server 2008 R2 with .NET Framework 4.7.2 .
+
+## Symptoms
+When you install the .NET Framework 4.7.2 on Windows 10 version 1803 or lower and your system was upgraded from Windows 7 SP1 with .NET Framework 4.7.2 , .NET 4.7.2 setup installation reports success but fails to install .NET 4.7.2 files in the machine.
+
+Consider the following scenario:
+  1. On a Windows 7 SP1 or Windows Server 2008 R2 , install the .NET Framework 4.7.2
+  2. Upgrade the machine to Windows 10 Fall Creators (version 1709) or lower. Upgraded OS can be any version of Windows 8.1 or Windows Server 2008 R2.
+  3. Run .NET 4.7.2 setup
+  4. .NET Framework 4.7.2 setup reports successful installation but fails to install framework files.
+
+Note: This issue does not occur when upgraded to Windows 10 April 2018 Update (version 1803) as it comes pre-installed with .NET Framework 4.7.2.
+
+## Cause
+This issue occurs because OS upgrade leaves behind .NET 4.7.2 setup file in Windows installer cache and its registration. Due to a bug in .NET setup that checks for the setup file in Windows installer and determines that .NET 4.7.2 is already installed and thus, skips installing .NET framework files on the upgraded Windows 10 machine.
+
+## Impact
+.NET 4.7.2 setup reports successful installation but skips installing its files and registries.
+
+Note: This issue occurs only when you do an upgrade from Windows 7 or Windows Server 2008 R2 with .NET 4.7.2 pre-installed.
+
+## Resolution
+Upgrade to latest Windows 10 April 2018 Update ( Version 1803), it comes pre-installed with .NET Framework 4.7.2 .
```

Comment:
- delete "latest"
- is upgrading to Windows 10 October 2018 Update also an alternative?
- instead of a comma, period before "it"
- it comes --> It comes (or "They come" if you add the October Update)
- no space after 2 and before the final period